ci_config.json

```diff
@@ -169,6 +169,18 @@
       "darwin": false,
       "windows": false
     },
+    "_comment": "- Upstream has no idea how to use feature options",
```

Comment:
Could you elaborate a bit what's the issue? Did you report upstream?

Reply:
IMHO, you should leave all options to `auto`, report upstream issue because it seems it disables some parts that should work by default. In ci-config.json we have that new section to tell some of deps provided are optional:
```
    "skip_dependency_check": [
      "libdrm_etnaviv",
      ...
    ]
```

Reply:
Makes no sense. the .wrap file specifies a bunch of dependencies that must be enabled.

Reply:
At least report issue upstream. Saying here they don't understand feature option and do nothing about it is not helping anyone.